content/general/authentication.textile

```diff
@@ -84,26 +84,26 @@ Whilst Basic Authentication is simple, we recommend it to be only used on the se
 
-Token authentication is the recommended authentication scheme for client devices as tokens are short-lived and may more readily be distributed to clients where there is a risk of compromise. Tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":#capabilities-explained or be limited to a "specific identity (<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>)":#identified-clients. Tokens are most commonly issued server-side and may be signed using a private API key which guarantees to Ably that the token request is authentic and is issued by a trusted issuer.
+Token authentication is the recommended authentication scheme for client devices as tokens are short-lived and may more readily be distributed to clients where there is a risk of compromise. Ably-compatible tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":#capabilities-explained or be limited to a "specific identity (<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>)":#identified-clients. Ably-compatible tokens are most commonly issued server-side, although the type of Ably-compatible token issued varies. At present, Ably supports "Ably Tokens":#token-process, "Ably TokenRequests":#token-request-process, "JWT with an embedded Ably Token":#jwt-embed-process, and "Ably JWT":#ably-jwt-process.
 
 Token authentication is the default authentication scheme when a client library is instanced with any of the following options:
 
 * a "<span lang="default">@token@</span><span lang="csharp">@Token@</span>":/realtime/types#client-options or "<span lang="default">@tokenDetails@</span><span lang="csharp">@TokenDetails@</span>":/realtime/types#client-options property is provided;
-* an "<span lang="default">@authUrl@</span><span lang="csharp">@AuthUrl@</span>":/realtime/types#client-options or "<span lang="default">@authCallback@</span><span lang="csharp">@AuthCallback@</span>":/realtime/types#client-options is provided that returns a token or token request;
+* an "<span lang="default">@authUrl@</span><span lang="csharp">@AuthUrl@</span>":/realtime/types#client-options or "<span lang="default">@authCallback@</span><span lang="csharp">@AuthCallback@</span>":/realtime/types#client-options is provided that returns an Ably-compatible token or Ably TokenRequest;
 * a "<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>":/realtime/types#client-options is provided;
 * "<span lang="default">@useTokenAuth@</span><span lang="csharp">@UseTokenAuth@</span>":/realtime/types#client-options is true
 
-Token authentication is typically done in one of two ways:
+Token authentication is typically done in one of four ways:
 
-h4. Signed token request is created by your servers and passed to clients
+h4(#token-request-process). Signed Ably TokenRequest is created by your servers and passed to clients
```

Comment:
I don't think we should use the term "signed TokenRequest", because it seems to suggest that there might be a kind of TokenRequest that's not signed. I would prefer that we say that a TokenRequest is signed where we define that term, and then elsewhere we just refer to a TokenRequest.